### ISSUE TITLE
Client should quit on SIGINT (ctrl-c) instead of ctrl-d (input EOF) on prompts?

### ISSUE THREAD

**Issue by gabriel** [keybase/go#121]
Super minor

**Comment by maxtaco** [comment]
Agreed, though I once tried and failed to get this to work.  @patrickxb, any ideas?

**Comment by patrickxb** [comment]
yeah, it's easy.  I'll do this.

**Comment by maxtaco** [comment]
what i got is in libcmdline/main.go in HandleSignals.

**Comment by patrickxb** [comment]
Hmmm...golang.org/x/crypto/ssh/terminal is capturing and discarding ctrl-c (and most unprintable keys)....ctrl-d it handles.

**Comment by maxtaco** [comment]
Ah I see.  We can potentially get off of ssh's terminal.  I was using it to handle password-entry and/or echoless input.  But we're using pinentry for that.  The question is, what happens if pinentry isn't installed?  We fall back to the terminal....

**Comment by patrickxb** [comment]
This might work:

https://github.com/Bowery/prompt

Supports password entry...
